chain/store/store.go

```diff
@@ -52,6 +58,72 @@ func NewLedgerStore() (*ChainStore, error) {
 	return chain, nil
 }
 
+func (cs *ChainStore) PrepareFastSync(syncRootHash common.Uint256, fastSyncHeight uint32) {
+	err := cs.persistSyncRootHeight(fastSyncHeight)
+	if err != nil {
+		log.Fatalf("Error persist sync root height: %v", err)
+	}
+	refCountHeightBuffer := make([]byte, 4)
+	binary.LittleEndian.PutUint32(refCountHeightBuffer[:], fastSyncHeight+1)
+	err = cs.st.BatchPut(db.TrieRefCountHeightKey(), refCountHeightBuffer)
```

Comment:
Should set ref count height and pruned height AFTER fast sync is done.
Also I think it should be -1 instead of +1, same for pruned height

Reply:
It would panic if we set this value after fast sync done when nknd restart, i need to test it to make sure